vcr/api/v1/api.go

```diff
@@ -89,13 +88,13 @@ func (w *Wrapper) Revoke(ctx echo.Context, id string) error {
 
 // Create a Verifiable credential
 func (w *Wrapper) Create(ctx echo.Context) error {
-	vc := did.VerifiableCredential{}
+	requestedVC := IssueVCRequest{}
```

Comment:
Is this change needed since it is an alias?

Reply:
Nope, but I wanted to use the right type since that's what's defined in the YAML spec